openupgrade_scripts/scripts/project/15.0.1.2/pre-migration.py

```diff
@@ -0,0 +1,79 @@
+from openupgradelib import openupgrade
+
+_column_renames = {
+    "project_task": [
+        ("user_id", None),
+    ],
+}
+
+
+def fill_project_project_allow_task_dependencies(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        ALTER TABLE project_project
+        ADD COLUMN allow_task_dependencies BOOLEAN""",
+    )
+    if not openupgrade.table_exists(env.cr, "task_dependencies_rel"):
+        openupgrade.logged_query(
+            env.cr,
+            """
+            UPDATE project_project
+            SET allow_task_dependencies = TRUE""",
+        )
+        return
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE project_project
+        SET allow_task_dependencies = TRUE
+        WHERE id in (
+            SELECT DISTINCT project_id as id
+            FROM project_task
+            WHERE id in (
+                SELECT DISTINCT task_id as id
+                FROM task_dependencies_rel
+            )
+        )""",
+    )
+
+
+def fill_project_project_last_update_status(env):
+    openupgrade.logged_query(
```

Comment:
Why not using here the openugradelib method for adding a field with such default?

Reply:
I didn't find a method in `openupgradelib` to do this.

Reply:
Done, using `add_fields` method.